### ISSUE TITLE
Issue with saveAsxxx() functions.

### ISSUE THREAD

**Issue by GoogleCodeExporter** [RoestVrijStaal/jmupdf#6]
```
All the saveAsXXX() functions are rotating the image by -1 degrees. This issue 
is found within the JNI code.

The solution to this will be to add a "rotate" parameter so that the Java code 
can control page rotation before saving image to disk.

This will be available in next minor release as a fix/enhancement.

The workaround is to change the "-1" value to zero.


```

Original issue reported on code.google.com by `pedro.ri...@gmail.com` on 8 Mar 2012 at 9:10

**Comment by GoogleCodeExporter** [comment]
```
The code has been fixed in revision 33c0edf0183a. New binaries that reflect 
this change will be available for v.0.4.2.
```

Original comment by `pedro.ri...@gmail.com` on 8 Mar 2012 at 10:03

**Comment by GoogleCodeExporter** [comment]
Original comment by `pedro.ri...@gmail.com` on 9 Mar 2012 at 4:44
- Changed state: **Fixed**
